crates/runtime/src/vmcontext.rs

```diff
@@ -434,6 +436,62 @@ impl VMGlobalDefinition {
         Self { storage: [0; 16] }
     }
 
+    /// Create a `VMGlobalDefinition` from a `ValRaw`.
+    ///
+    /// # Unsafety
+    ///
+    /// This raw value's type must match the given `WasmValType`.
+    pub unsafe fn from_val_raw(wasm_ty: WasmValType, raw: ValRaw) -> Self {
+        let mut global = Self::new();
+        match wasm_ty {
+            WasmValType::I32 => *global.as_i32_mut() = raw.get_i32(),
+            WasmValType::I64 => *global.as_i64_mut() = raw.get_i64(),
+            WasmValType::F32 => *global.as_f32_bits_mut() = raw.get_f32(),
+            WasmValType::F64 => *global.as_f64_bits_mut() = raw.get_f64(),
+            WasmValType::V128 => *global.as_u128_bits_mut() = raw.get_v128().to_ne_bytes(),
```

Comment:
If this `.to_ne_bytes()` is required I think that means we're doing something wrong in `ValRaw`, perhaps a missing `from_le_bytes` in `get_v128`?

No matter what if this is the only one that does `.to_ne_bytes()` is should have docs as to why it's omitted on the others.

Reply:
I was just doing this to get a `[u8; 16]` but it turns out that isn't necessary: we have a method on `wasmtime_runtime::Global` to get the value as a `&mut u128`, which I didn't see before.